propagation.go

```diff
@@ -128,21 +131,27 @@ func extractTraceContext(opaqueCarrier interface{}) (SpanContext, error) {
 		pickupW3CTraceContext(http.Header(c), &spanContext)
 	}
 
+	// Iterate over the headers, look for Instana headers and try to parse them.
+	// In case of error interrupt, iteration and return it.
 	err := carrier.ForeachKey(func(k, v string) error {
 		var err error
 
 		switch strings.ToLower(k) {
 		case FieldT:
 			spanContext.TraceIDHi, spanContext.TraceID, err = ParseLongID(v)
 			if err != nil {
+				sensor.logger.Debug("extract trace context ", FieldT, ": ", err)
```

Comment:
I guess this debug log is intended for debugging the case when headers are broken? In this case it might make sense to include the original headers into the same message, otherwise there is a high chance these two lines will be impossible find in the log.

Reply:
Agree 👍  I add it.